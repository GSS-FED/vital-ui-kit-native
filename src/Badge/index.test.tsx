import { create as renderer } from 'react-test-renderer';
import * as createContext from 'react-cosmos-test/generic';
import fixture from './index.fixture';

const { mount, getWrapper } = createContext({ renderer, fixture });

beforeEach(mount);

test('matches snapshot', () => {
  expect(getWrapper().toJSON()).toMatchSnapshot();
});
