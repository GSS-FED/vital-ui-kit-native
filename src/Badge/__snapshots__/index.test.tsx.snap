// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#D8D8D8",
        "borderRadius": 7.5,
        "height": 16,
        "paddingLeft": 7,
        "paddingRight": 7,
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#3D414A",
          "fontSize": 12,
        },
        undefined,
      ]
    }
  >
    badge
  </Text>
</View>
`;
